contracts/Vault.sol

```diff
@@ -9,13 +9,13 @@ import {ERC165} from "@openzeppelin/contracts/utils/introspection/ERC165.sol";
 import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
 import {ReentrancyGuard} from "@openzeppelin/contracts/security/ReentrancyGuard.sol";
 import {Pausable} from "@openzeppelin/contracts/security/Pausable.sol";
+import {Counters} from "@openzeppelin/contracts/utils/Counters.sol";
```

Comment:
We removed Counters a while ago because it wasn't doing anything. Feels like it's also unnecessary in this situation, right?

Reply:
I thought it saved gas by skipping using safemath overflow checks?

Reply:
@naps62 u know?

Reply:
yes, that's one of the suposed benefits (you could use `unchecked` yourself, but then you'd make your own code less readable)
I never actually measured if it's any cheaper, though. I assume it is, and I assume those functions are inlined by the compiler. Probably worth doing a quick benchmark